standard/sections/clause_7_normative_text.adoc

```diff
@@ -77,31 +77,32 @@ The table below provides an overview of the set of properties that may be includ
 
 |``links``
 |**required**
-|Online linkages to data retrieval or additional resources associated with the dataset (see <<_links>>)
+|Online linkages for data retrieval or additional resources associated with the dataset (see <<_links>>)
 
 |===
 
 
 ==== Message size
 
-The WIS2 Notification Message allows for transmission of messages in a compacted or minified manner, and includes the ability to embedding content inline as required (see <<Properties / Content>>).
+The WIS2 Notification Message allows for transmission of messages in a compact manner, and includes the ability to embedding content inline as required (see <<Properties / Content>>).
 
 include::../requirements/core/REQ_message_size.adoc[]
 
-==== GeoJSON compliance
+//==== GeoJSON compliance
 
-The WIS2 Notification Message schema is based on _GeoJSON_ (RFC7946) and its associated information model.  Compliant messages are therefore compliant with _GeoJSON_.
+//The WIS2 Notification Message schema is based on _GeoJSON_ (RFC7946) and its associated information model.  Compliant messages are therefore compliant with _GeoJSON_.
 
-include::../requirements/core/REQ_validation.adoc[]
+//include::../requirements/core/REQ_validation.adoc[]
 
```

Comment:
I removed the GeoJSON compliance section, because all of the GeoJSON properties are described below in other sections.

Reply:
We should keep for clarity.

Reply:
ok